src/osx/carbon/font.cpp

```diff
@@ -307,9 +314,18 @@ void wxFontRefData::MacFindFont()
         wxString lookupnameWithSize = wxString::Format( "%s_%u_%d", m_info.m_faceName, traits, m_info.m_pointSize );
 
         static std::map< std::wstring , wxCFRef< CTFontRef > > fontcache ;
+        static std::map< std::wstring , wxCFRef< CFDictionaryRef > > fontattributescache ;
```

Comment:
Looking at how this map is used, it seems it would be better to have a single map of font name+size to pair/struct of font and attributes refs. I.e. why not change `fontcache` value type instead of adding another map?